### PR TITLE
configure python universal wheels Refs #115

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

**Comment by graingert** [comment]
this will also create a .whl file which will need to be uploaded to pypi

File: GNUmakefile [hunk 1]
```diff
@@ -432,7 +432,7 @@ dist_libstemmer_python: $(PYTHON_SOURCES)
 	cp -a $(PYTHON_SAMPLE_SOURCES) $${dest}/src/$(python_sample_dir) && \
 	cp -a $(PYTHON_RUNTIME_SOURCES) $${dest}/src/$(python_runtime_dir) && \
 	cp -a $(PYTHON_PACKAGE_FILES) $${dest} && \
-	(cd $${dest} && $(python) setup.py sdist && cp dist/*.tar.gz ..) && \
+	(cd $${dest} && $(python) setup.py sdist bdist_wheel && cp dist/*.{tar.gz,whl} ..) && \
```
Comment (graingert) [inline GNUmakefile]:
```suggestion
	(cd $${dest} && $(python) setup.py sdist bdist_wheel && cp dist/snowballstemmer-*{.tar.gz,-py2.py3-none-any.whl} ..) && \
```

Comment (graingert) [reply GNUmakefile]: might be better to be more specific with the dist filenames

**Comment by graingert** [comment]
@ojwb

File: GNUmakefile (hunk 1 above)
Comment (mitya57) [inline GNUmakefile]: The syntax with curly brackets is a bashism. It won't work when make uses another shell (e.g. dash).

```suggestion
	(cd $${dest} && $(python) setup.py sdist bdist_wheel && cp dist/*.tar.gz dist/*.whl ..) && \
```

File: python/setup.cfg [hunk 2]
```diff
@@ -0,0 +1,2 @@
+[bdist_wheel]
+universal=1
```
Comment (mitya57) [inline python/setup.cfg]: You need to also add this file to `PYTHON_PACKAGE_FILES` in GNUmakefile.

File: python/setup.py [hunk 3]
```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env python
 
-from distutils.core import setup
+try:
+    from setuptools import setup
+except ImportError:
+    from distutils.core import setup
```
Comment (mitya57) [inline python/setup.py]: Is this fallback really needed? Most Python modules just use setuptools nowadays.

Comment (graingert) [reply python/setup.py]: @mitya57 almost certainly not needed anymore, but most people who still use `distutils.core.setup` are ideologically opposed to removing it as a fallback

Comment (mitya57) [reply python/setup.py]: I think distutils was here only for historical reasons.

**Comment by ojwb** [comment]
> @ojwb

Contentless pings are just annoying.  If you really want my attention, at least go to the effort of telling me why.

@mitya57 Are you happy with this now?

**Comment by mitya57** [comment]
Yes, looks good to me now.

**Pull request merged**